app/views/simulations/index.html.erb

```diff
@@ -1,22 +1,29 @@
-<table>
-    <thead>
-        <tr>
-            <th>Simulation</th>
-            <th>Name</th>
-            <th>Runs</th>
-            <th>Started</th>
-            <th>Finished</th>
-        </tr>
-    </thead>
-    <tbody>
-        <% @simulations.each do |simulation| %>
-            <tr>
-                <td><%= simulation.id %></td>
-                <td><%= link_to(simulation.name, simulation) %></td>
-                <td><%= simulation.runs %></td>
-                <td><%= simulation.start %></td>
-                <td><%= simulation.finish %></td>
-            </tr>
-        <% end %>
-    </tbody>
-</table>
+<div >
+    <div class="py-4">
+        <%= link_to "New simulation", new_simulation_path, class: "bg-blue-800 text-white px-5 py-2 rounded-full hover:bg-blue-400" %>
+    </div>
+    <div>
+        <table>
+            <thead>
+                <tr>
+                    <th>Simulation</th>
+                    <th>Name</th>
+                    <th>Runs</th>
+                    <th>Started</th>
+                    <th>Finished</th>
+                </tr>
+            </thead>
+            <tbody>
+                <% @simulations.each do |simulation| %>
+                    <tr>
+                        <td><%= simulation.id %></td>
+                        <td><%= link_to(simulation.name, simulation, class:"underline hover:font-semibold") %></td>
+                        <td><%= simulation.runs %></td>
+                        <td><%= simulation.start %></td>
+                        <td><%= simulation.finish %></td>
+                    </tr>
+                <% end %>
```

Comment:
You can also add an empty state here for when `@simulations.none?`, if you want more Tailwind practice. Here's some code that may be of use:

```html
<div class="text-center">
  <svg class="mx-auto h-12 w-12 text-gray-400" fill="none" viewBox="0 0 24 24" stroke="currentColor" aria-hidden="true">
    <path vector-effect="non-scaling-stroke" stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M9 13h6m-3-3v6m-9 1V7a2 2 0 012-2h6l2 2h6a2 2 0 012 2v8a2 2 0 01-2 2H5a2 2 0 01-2-2z" />
  </svg>
  <h3 class="mt-2 text-sm font-semibold text-gray-900">No projects</h3>
  <p class="mt-1 text-sm text-gray-500">Get started by creating a new project.</p>
  <div class="mt-6">
    <button type="button" class="inline-flex items-center rounded-md bg-indigo-600 px-3 py-2 text-sm font-semibold text-white shadow-sm hover:bg-indigo-500 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-indigo-600">
      <svg class="-ml-0.5 mr-1.5 h-5 w-5" viewBox="0 0 20 20" fill="currentColor" aria-hidden="true">
        <path d="M10.75 4.75a.75.75 0 00-1.5 0v4.5h-4.5a.75.75 0 000 1.5h4.5v4.5a.75.75 0 001.5 0v-4.5h4.5a.75.75 0 000-1.5h-4.5v-4.5z" />
      </svg>
      New Project
    </button>
  </div>
</div>
```

Reply:
(It's a Tailwind template that looks like this:

<img width="1154" alt="image" src="https://github.com/eargollo/soccer/assets/1158152/7ddc89e6-1e59-4ea7-9b10-cb80447b6fa6">

Reply:
Neat. Added on #21